terraform/production/main.tf

```
@@ -237,6 +237,10 @@ module "marklogic" {
     local.all_notifications_email_addresses,
     ["deltastatsupport@communities.gov.uk"]
   )
+  s151_job_notification_emails = concat(
+    local.all_notifications_email_addresses,
+    ["deltastatsupport@communities.gov.uk"]
+  )
```

I don't think we need to notify stats support here.
```suggestion
  s151_job_notification_emails = concat(
    local.all_notifications_email_addresses
  )
```